source/dyaml/loader.d

```diff
@@ -72,6 +73,15 @@ struct Loader
                                         .format(filename, e.msg));
             }
          }
+         /// Ditto
+         static Loader fromFile(File file) @system
+         {
+            // Use a reasonable chunk size for reading
+            enum chunkSize = 4096;
+            auto loader = Loader(file.byChunk(chunkSize).join);
```

Comment:
you can put the integer literal directly as parameter, comment has no added value.